src/client/src/routes/StyleguideRoute/StyleguideRoute.tsx

```diff
@@ -0,0 +1,73 @@
+import _ from 'lodash'
+import React from 'react'
+import { BrowserRouter, Route, Switch } from 'react-router-dom'
+import { styled, ThemeState } from 'rt-theme'
+
+import FloatingTools from './components/FloatingsTools'
+import { Block, SectionBlock } from './styled'
+
+import Atoms from './sections/Atoms'
+import ColorSpectrum from './sections/ColorSpectrum'
+import CoreBranding from './sections/CoreBranding'
+import FontFamilies from './sections/FontFamilies'
+import Introduction from './sections/Introduction'
+
+const sections = _.mapKeys(
```

Comment:
There is a lot of dependency on lodash in this file. Are there any areas where it might be excessive?

Reply:
There is no meaningful cost due to the use of `babel-lodash-plugin` — and no, I don't think use of a functional tool like this is excessive.

Additionally, the tooling around `lodash` is some of the best — therefore it shouldn't impose any costs on the developer. And, while the API should be obvious simply by convention it also comes with complete documentation in VSCode or Atom with an extension.

There is every reason to use it where it provides a relevant functional primitive that simplifies the expression and adds no cognitive or runtime overhead.